clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1369,6 +1369,61 @@ def paths_to_bids(
     return output_file_treated
 
 
+def _get_images_with_suffix(
+    folder: Path,
+    suffixes: Iterable[str],
+) -> Iterable[Path]:
+    import re
+
+    if suffixes:
+        rgx = re.compile(r"|".join(suffixes))
+        images = list(filter(rgx.search, [f.name for f in folder.iterdir()]))
+        return [folder / image for image in images]
+    return []
+
+
+def _remove_existing_images_if_necessary(
+    folder: Path,
+    suffixes: Iterable[str],
+    mod_to_update: bool = False,
+) -> bool:
```

Comment:
No worries ! The thing is, I can't use the output of that function by itself if I want to get out of the `create_file` function since it's totally possible for it to return either an empty or an actual list while we need the process to continue. 
If we want to get the images_to_remove list we could call `_get_images_with_suffix` separately and keep a function returning a flag to know if we continue the process. WDYT ?

Reply:
Indeed, you're totally right !
Let's forget about this comment and keep things like this.
Sorry for the noise.

Reply:
Did you commit the suggestions on the tests ? I see the conversations resolved but not the commits and modifications.

Reply:
My bad, I was waiting for your answer before committing the other changes. You should see them now @NicolasGensollen